Makefile

```diff
@@ -0,0 +1,39 @@
+# Copyright 2017 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+.PHONY: all rbdplugin
+
+IMAGE_NAME=192.168.80.240:4000/csi_images/rbdplugin
```

Comment:
why this IP?

Reply:
For testing I am using internal docker registry where I automatically push the image, should I change it for a local docker image?

Reply:
you can set default registry in docker config via `--add-registry`. On Fedora, it is in /etc/sysconfig/docker

Reply:
please remove private docker registry from code